.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+## Proposed changes
+
+Include description and references to tickets or issues that this PR is related to
+
+## Checklist prior to adding reviewers
+
+_Put an `x` in the boxes that apply._
+
+- [ ] I have run this successfully locally using `./build.ps1 compose`
+- [ ] I have added tests for my code and/or updated existing tests where my changes impacts them, or there is a valid explanation for not including tests in this PR
+- [ ] I have added necessary documentation (if appropriate)
+- [ ] I created this branch from main and have integrated the latest changes
+- [ ] I conformed to the [Angular Commit Message Format](https://gist.github.com/brianclements/841ea7bffdb01346392c#commit-message-format) for all commit messages in this PR
```

Comment:
This is not the pattern. We use plain English, third person singular in present tense across the board - e.g. *fixes ...* or *integrates ...*  and include the GitHub issue number when one is available. 

Ideally, all changes should be logged in GitHub issues so that they can be linked to a commit / PR.

Reply:
So we use Angular Commit Message Formatting as it gives a clear indication of the type of change being made and encourages developers to apply the principal of small incremental change to the project.

Although not specified here in this list of PR Request requirements, the branch should include the reference to the issue being resolved. At present, the project is using an external Jira instance to track change and not GitHub issues. Clarification on which system to track change it appears is required.

The additional benefit of using the Angular convention is that we can map change types to different use cases when it comes to semantic versioning. We can leverage different tasks available to automatically derive the correct version string based on the commit history.